resource/temporary-resource-storage/src/main/kotlin/com/ritense/resource/autoconfigure/TemporaryResourceStorageAutoConfiguration.kt

```diff
@@ -44,4 +47,19 @@ class TemporaryResourceStorageAutoConfiguration {
         return TemporaryResourceStorageDeletionService(retentionInMinutes)
     }
 
+    @Bean
+    @ConditionalOnMissingBean(TemporaryResourceStorageResource::class)
+    fun temporaryResourceStorageResource(
```

Comment:
We definitely need more 'resource' in this name.

Reply:
We also have a `ResourceResource` :p